cogeo_mosaic/backends/base.py

```diff
@@ -60,6 +60,11 @@ def quadkey_zoom(self) -> int:
         """Return Quadkey zoom property."""
         return self.mosaic_def.quadkey_zoom or self.mosaic_def.minzoom
 
+    @property
+    def quadkeys(self) -> List[str]:
+        """Return the list of quadkey tiles."""
+        return [qk for qk, _ in self.mosaic_def.tiles.items()]
```

Comment:
You can just do `list(self.mosaic_def.tiles.keys())`.

I think that's clearer than a list comprehension here